sample-configuration.yml

```diff
@@ -1,9 +1,11 @@
 stations:
   - id: EVB-P17390866
+    basicAuthPassword: station_password
```

Comment:
can we default it to 40x F in case it wasn't passed in the command line?

Reply:
This file does not contain default values. Its a sample of configuration. Whats so special about this magic number? :)

Reply:
It's often a default password and it would be quite annoying to put it to a command line every time.

Reply:
ok, never seen such passwords, but anyway will do